translations/player_turns.tex/de.po

```diff
@@ -229,6 +314,14 @@ msgid ""
 "As Alice announces her intent to start Combat, both players still have an opportunity to perform Town Actions.}\\par\n"
 "\n"
 msgstr ""
+"\\textit{Alicia spielt mit Schlossheldin Catherine und beginnt ihren Zug.\n"
+"Sie hat 3 Karten aus der vorherigen Runde auf der Hand und möchte 2 davon abwerfen, bevor sie bis zum \\mbox{Handkartenlimit \\svg{hand}} nachzieht.\n"
+"Ihr Handkartenlimit ist 5, da ihr Hauptheld auf Stufe 3 ist. Also zieht sie nach dem Abwerfen noch 4 Karten (siehe \\pagelink{Level}{Stufeneffekte}).}\\par\n"
+"\\textit{Anschließend nutzt sie zuerst den Baumarker, um die \\svgunit{golden} Behausung zu bauen, und danach den Bevölkerungsmarker, um die \\svgunit{golden} Einheit Kämpe zu \\pagelink{Units}{rekrutieren}.\n"
+"Sie darf dies nur tun, da sie die niederen Behausungen (\\svgunit{bronze} und \\svgunit{silver}) bereits errichtet hat und genügend \\pagelink{Resources}{Ressourcen} für den Bau der Behausung und die Rekrutierung der Kämpen besitzt.}\\par\n"
+"\\textit{Auf diese Weise für den Kampf vorbereitet gibt sie einen BP aus, um ihren Helden auf das angrenzende Feld zu bewegen, welches vom Haupthelden Sandro ihres Mitspielers Bob besetzt ist.\n"
+"Alicia und Bob haben jetzt noch Gelegenheit weitere Stadtaktionen durchzuführen. Alicia hat aber bereits gebaut und rekrutiert und dürfte nur noch Zauber kaufen.}\\par\n"
```

Comment:
```suggestion
"\\textit{Vorbereitet auf den Kampf gibt sie einen BP aus, um ihren Helden auf das angrenzende Feld zu bewegen, das vom Haupthelden Sandro ihres Mitspielers Bob besetzt ist.\n"
"Alicia und Bob haben noch Gelegenheit, weitere Stadtaktionen durchzuführen. Alicia hat jedoch bereits gebaut und rekrutiert und darf nur noch Zauber kaufen.}\\par\n"
```

Reply:
Will get rid of "kaufen" orphan.

Reply:
> Will get rid of "kaufen" orphan.

Its done! Thanx for your suggestions! :)